pkg/cmd/secrets.go

```diff
@@ -590,13 +592,15 @@ func init() {
 	secretsCmd.Flags().StringP("project", "p", "", "project (e.g. backend)")
 	secretsCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	secretsCmd.Flags().Bool("raw", false, "print the raw secret value without processing variables")
+	secretsCmd.Flags().Bool("vistypes", false, "when printing the table, include secret visibility types")
 	secretsCmd.Flags().Bool("only-names", false, "only print the secret names; omit all values")
 
 	secretsGetCmd.Flags().StringP("project", "p", "", "project (e.g. backend)")
 	secretsGetCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	secretsGetCmd.Flags().Bool("plain", false, "print values without formatting")
 	secretsGetCmd.Flags().Bool("copy", false, "copy the value(s) to your clipboard")
 	secretsGetCmd.Flags().Bool("raw", false, "print the raw secret value without processing variables")
+	secretsGetCmd.Flags().Bool("vistypes", false, "when printing the table, include secret visibility types")
```

Comment:
Unfortunately you'll also need to add these flags to our legacy `doppler enclave secrets` command (`enclave_secrets.go`) or they'll error

Reply:
Ah, that's why the test was failing. Will add, thanks!